### ISSUE TITLE
add lookup icon to mobile navbar

### ISSUE THREAD

**Issue by balmas** [alpheios-project/alpheios_nemo_ui#182]
@abrasax requests that we separate out the lookup and tools functionalities in the mobile reader to be consistent with the way the functionality is offered on the desktop in the floating toolbar.

This depends upon alpheios-project/components#607 alpheios-project/components#607 and alpheios-project/embed-lib#62

the lookup icon should come immediately before the help icon

**Comment by balmas** [comment]
@kirlat this (together with related issues in components and embed-lib) is next priority after alpheios-project/node-build#16

**Comment by balmas** [comment]
@kirlat can you make this change too now that the enabling changes in components and embed-lib are done? Thanks!

**Comment by kirlat** [comment]
Sure, will do that

**Comment by balmas** [comment]
reading for testing

**Comment by monzug** [comment]
Basically we are back to the "floating toolbar" on mobile too, correct? see screenshot how it looks like on iPhone xr.
can we resize the "new" toolbar and reduce some extra space on the top?

![IMG_3754](https://user-images.githubusercontent.com/41396793/61957608-a4b3b100-af8d-11e9-876c-231a71b10069.PNG)

**Comment by balmas** [comment]
apparently, yes, it's the floating toolbar, without the icon that opens it anchored.  Agree we need to remove the extra space at the top. Back to @kirlat

**Comment by balmas** [comment]
also, per discussion with @monzug we think that opening the lookup or toolbar from the mobile nav menu should close the panel. This is a difference in behavior from the desktop that we would want *only for mobile* where there isn't really room for both to be open at the same time.  @kirlat how difficult do you think that will be to implement?

**Comment by monzug** [comment]
uhm, I don't see much change with the previous look of the toolbar. also panel is still open and lookup is selectable but hidden by the panel

**Comment by balmas** [comment]
ah, I mistakenly gave this to you to verify, thinking the changes were made. Will take it back. Sorry!

**Comment by monzug** [comment]
no problem. I thought it was my connection so I kept moving around the apt
with no much luck...

**Comment by balmas** [comment]
ok, this really is ready for testing now, but note that the changes are a little different.

In looking at closing the panel open opening  the lookup and toolbar, I realized we had some bugs in the way open/closed state of the lookup and toolbar were handled in the text environment because we were trying to manage that state in two places -- the core libraries and the text app code. Getting that right was going to be too complex, so now instead of the tools and lookup button in the mobile texts trying to act as toggles to open or close the toolbar and lookup, they will always open them, and now the close icon is back on the toolbar and lookup. This means we needed the extra space on the toolbar to make room for the close icon, although I did generally try to clean up the spacing as much as possible.

I did also implement the change to close the panel when the lookup or toolbar are open.
